src/lib/markbind/src/lib/markdown-it-shared/markdown-it-icons.js

```diff
@@ -1,15 +1,22 @@
 const octicons = require('@primer/octicons');
 
 module.exports = require('markdown-it-regexp')(
-    /:(fa[brs]|glyphicon|octicon)-([a-z-]+):/,
+    /:(fa[brs]|glyphicon|octicon|octiconlight)-([a-z-]+)~?([a-z-]+)?:/,
     (match, utils) => {
         let iconFontType = match[1];
         let iconFontName = match[2];
+        let iconClass = match[3];
 
         if (iconFontType === 'glyphicon') {
             return `<span aria-hidden="true" class="glyphicon glyphicon-${iconFontName}"></span>`;
         } else if (iconFontType === 'octicon') {
+          if (iconClass) {
+            return octicons[iconFontName].toSVG({"class": iconClass});
```

Comment:
I personally prefer ternary expressions instead of 1-line `if-else`s especially since this is nested, but i'm fine eitherway 😄